ci/v2/lint.sh

```diff
@@ -16,28 +16,19 @@ lint_readme() {
   fi
 }
 
-lint_scripts() {
+scripts() {
```

Comment:
what type of scripts? maybe rename to `shell_scripts`?

Reply:
changed to code_format, I think that's what it is (shell, python, java, cpp)